src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -383,13 +383,16 @@
     producer,
     voltage_per_phase,
 )
+from ._power_managing import PowerManagerAlgorithm
 
 
 async def initialize(
     server_url: str,
     resampler_config: ResamplerConfig,
     *,
     api_power_request_timeout: timedelta = timedelta(seconds=5.0),
+    # pylint: disable-next: line-too-long
+    battery_power_manager_algorithm: PowerManagerAlgorithm = PowerManagerAlgorithm.SHIFTING_MATRYOSHKA,  # noqa: E501
```

Comment:
I guess this is needed somehow soon, but in the future I think it might make more sense to have copy what we have with the `resampler_config and add a `power_manager_config: PowerManagerConfig`, and in there it can have something like `power_manager_config.battery.algorithm` for example.